LICENSE-binary

```diff
@@ -218,6 +218,7 @@ javax.jdo:jdo-api
 joda-time:joda-time
 net.sf.opencsv:opencsv
 org.apache.derby:derby
+org.ehcache:ehcache
```

Comment:
ehcache is ALv2, good; I think there's a NOTICE entry for it though:
https://github.com/ehcache/ehcache3/blob/master/NOTICE
(Not sure what version this is pulling in; this is 3.x)

Reply:
Version is right.  3.3.1.  The NOTICE file is not under META-INF, miss this file, thanks a lot.